csv-files/readers.csv

```diff
@@ -0,0 +1,6 @@
+name,email,password
+Robert Baratheon,ours_is_fury@gmail.com,password123
+Arya Stark,winter_is_comming@gmail.com,password123
+Jamie Lanister,pays_debts@gmail.com,password123
+Loris Tyrell,flower_strong@gmail.com,password123
+Jon Snow,knows_nothing@gmail.com,password123
```

Comment:
What is the purpose of this file? If it's not needed for the final project it should be deleted